ui/admin/app/controllers/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts/index.js

```diff
@@ -0,0 +1,47 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+
+export default class ScopesScopeHostsHostCatalogsHostCatalogHostSetsHostSetHostsController extends Controller {
+  // =services
+  @service intl;
+
+  // =attributes
+  /**
+   * Returns the static columns header labels for the table
+   * @type {Array.<object>}
+   */
+  get isStaticColumns() {
```

Comment:
These getters could have just been one getter that does the `isStatic` check in it instead of the template. And they shouldn't be named "`is` + Static/DynamicColumns" since they don't return booleans.

Reply:
@ZedLi good call out, working on it now